internal/adsysservice/policy.go

```diff
@@ -10,11 +10,13 @@ import (
 	"github.com/ubuntu/adsys/internal/authorizer"
 	log "github.com/ubuntu/adsys/internal/grpc/logstreamer"
 	"github.com/ubuntu/adsys/internal/i18n"
+	"github.com/ubuntu/adsys/internal/policies"
 	"github.com/ubuntu/decorate"
 	"golang.org/x/sync/errgroup"
 )
 
 // UpdatePolicy refreshes or creates a policy for current user or user given as argument.
+// It can purge the policy instead of updating it if requested.
 func (s *Service) UpdatePolicy(r *adsys.UpdatePolicyRequest, stream adsys.Service_UpdatePolicyServer) (err error) {
```

Comment:
I’m starting to think that it may be more justifiable to have a dedicated APIs to purge, rather than trying to reuse it and skip the download in updatePolicyFor.

This way, it’s a separate API, defined, under the policy umbrella.

Reply:
I am a little bit less concern about the API here, depending on how complex or not the copy is. I think thought that we should just pass empty policies to the policy manager itself.

(I liked the fact that you add a separate user command still, rather than an argument which drastically changes the behaviour)

Reply:
> You mean directly from this UpdatePolicy function? 
No, I mean, I thought a separate API would have been better, but if you don’t think so, the most important part to me is that `ApplyPolicies` don’t know about the purge option itself.